### PR TITLE
4253 sendinblue fix prop to

### PR DESCRIPTION
**Pull request by alysonturing**
## SendinBlue

Fixing the prop "To" in send-transactional-email
Adding missing package.json 

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4274"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/sendinblue/actions/send-transactional-email/send-transactional-email.mjs [hunk 1]
```diff
@@ -3,8 +3,8 @@ import sendinBlueApp from "../../sendinblue.app.mjs";
 export default {
   key: "sendinblue-send-transactional-email",
   name: "Send Transactional Email",
-  description: "Send transactional email. [See the docs here](https://developers.sendinblue.com/reference/sendtransacemail)",
-  version: "0.0.3",
+  description: "Send transactional email. [See the docs](https://developers.sendinblue.com/reference/sendtransacemail) for more information.",
+  version: "0.0.39",
```
Comment (andrewjschuang) [inline components/sendinblue/actions/send-transactional-email/send-transactional-email.mjs:7]:
```suggestion
  version: "0.0.4",
```

**Comment by DilanAthukorala** [comment]
Verification completed, this is ready for release.

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**